src/__tests__/resources/events.integration.spec.ts

```diff
@@ -192,6 +192,17 @@ describe('Events integration test', () => {
       expect(items.length).toBeGreaterThan(0);
       expect(items[0].endTime).toBeUndefined();
     });
+
+    test('activeAtTime', async () => {
+      const { items } = await client.events.list({
+        filter: {
+          activeAtTime: { min: 101 },
+        },
+        limit: 10,
+      });
+
+      expect(items.length).toBe(1);
```

Comment:
.toBeGreaterThan(0) would be enough

Reply:
in case someone adds some random event

Reply:
Ok. I can do that and revert https://github.com/cognitedata/cognite-sdk-js/pull/365/commits/fa6802afca6a8e5962708b190adfcbf67d972b03 if you want

Reply:
https://github.com/cognitedata/cognite-sdk-js/commit/fa6802afca6a8e5962708b190adfcbf67d972b03 is fine

Reply:
I just though you finished)

Reply:
I reverted back. Test failed due to eventual consistency